early-christianity.csl

```diff
@@ -0,0 +1,792 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="chicago" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Early Christianity</title>
+    <title-short>EC</title-short>
+    <id>http://www.zotero.org/styles/early-christianity</id>
+    <link href="http://www.zotero.org/styles/early-christianity" rel="self"/>
+    <link href="http://www.zotero.org/styles/society-of-biblical-literature-fullnote-bibliography" rel="template"/>
+    <link href="https://www.mohr.de/en/journals/early-christianity-ec#menu4" rel="documentation"/>
+    <author>
+      <name>Carl Johan Berglund</name>
+      <email>cajo@cajo.nu</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="theology"/>
+    <issn>1868-7032</issn>
+    <issn>1868-8020</issn>
+    <summary>Early Christianity</summary>
+    <updated>2015-08-25T09:08:29+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="sv-SE">
```

Comment:
can you submit in Swedish? I thought it was just English in German. If I'm right about this, please remove this locale (down to line 32).